pandas/core/groupby/groupby.py

```diff
@@ -1020,6 +1020,10 @@ def curried(x):
                 return self.apply(curried)
 
             is_transform = name in base.transformation_kernels
+
+            if is_transform and self._obj_with_exclusions.empty:
```

Comment:
Was this part of testing https://github.com/pandas-dev/pandas/pull/47840? i.e. should this change be here?

Reply:
Yea - that's exactly it. Thanks.